src/WarcraftLegacies.Source/Quests/Scourge/QuestCultoftheDamned.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.ObjectiveSystem.Objectives.TimeBased;
+using MacroTools.QuestSystem;
+
+namespace WarcraftLegacies.Source.Quests.Scourge
+{
+  public sealed class QuestCultoftheDamned : QuestData
+  {
+
+    public QuestCultoftheDamned() : base("The Cult of the Damned",
+      "The Kingdom of Lordaeron must be eliminated to pave the way for the Legion's arrival.",
```

Comment:
This flavour doesn't make sense. The quest has nothing to do with the Legion's arrival, just the establishment of the Cult.